src/cdk/a11y/aria-describer/aria-describer.ts

```diff
@@ -186,6 +186,7 @@ export class AriaDescriber implements OnDestroy {
       messagesContainer = this._document.createElement('div');
       messagesContainer.id = MESSAGES_CONTAINER_ID;
       messagesContainer.classList.add('cdk-visually-hidden');
+      messagesContainer.style.visibility = 'hidden';
```

Comment:
I would set `visibility: hidden` before applying `cdk-visually-hidden`, and then add a comment on the `classList.add` call like
```ts
// Even though we use `visibility: hidden`, we still apply `cdk-visually-hidden`
// so that the description element doesn't impact page layout.
```